build/ci/save_driver_pods_logs.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash -x
 
-kubectl logs $(kubectl get pod -l csi | grep controller | awk '{print$1}') -c ibm-block-csi-controller > /tmp/driver_$(kubectl get pod -l csi | grep controller | awk '{print$1}')_logs.txt
-kubectl logs $(kubectl get pod -l csi | grep node | awk '{print$1}') -c ibm-block-csi-node > /tmp/driver_$(kubectl get pod -l csi | grep node | awk '{print$1}')_logs.txt
-kubectl logs $(kubectl get pod -l csi | grep operator | awk '{print$1}') > /tmp/driver_$(kubectl get pod -l csi | grep operator | awk '{print$1}')_logs.txt
+get_pod_logs_or_events (){
+    pod_name=$1
+    information_type=$2
+    kubectl $information_type $(kubectl get pod -l csi | grep $pod_name | awk '{print$1}') -c ibm-block-csi-$pod_name > /tmp/driver_$(kubectl get pod -l csi | grep $pod_name | awk '{print$1}')_$information_type.txt
```

Comment:
> $(kubectl get pod -l csi | grep $pod_name | awk '{print$1}')

It should be saved in a local variable since we use it twice and the entire command is too complicated anyway.

Reply:
done

Reply:
done